app/controllers/monsters_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MonstersController < ApplicationController
+
+  def index
+    @monsters = Monster.all
+  end
+        
+end
```

Comment:
Looking down, I actually see what you did! Its not a bad implementation. You just have this controlelr render that ERB file we need for react and have it take over from there. I would allow this, but recommend still what I suggest in your routes.rb file